src/platform/x11.rs

```diff
@@ -1,6 +1,8 @@
 use std::os::raw;
 use std::{ptr, sync::Arc};
 
+#[cfg(feature = "x11")]
```

Comment:
You can remove these now

Reply:
True that. Fixed.

Reply:
This... Does not seem fixed to me, could you take a second look?

Reply:
Sorry... I saw a different line... Now removed that `#cfg[...]`